modules/nf-core/canu/meta.yml

```diff
@@ -0,0 +1,72 @@
+name: "canu"
+description: Accurate assembly of segmental duplications, satellites, and allelic variants from high-fidelity long reads.
+keywords:
+  - Assembly
+  - pacbio
+  - hifi
+  - nanopore
+tools:
+  - "canu":
+      description: "Canu is a fork of the Celera Assembler designed for high-noise single-molecule sequencing."
+      homepage: "https://canu.readthedocs.io/en/latest/index.html#"
+      documentation: "https://canu.readthedocs.io/en/latest/tutorial.html"
+      tool_dev_url: "https://github.com/marbl/canu"
+      doi: "doi:10.1101/gr.215087.116"
+      licence: "['GPL v2 and others']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - reads:
+      type: file
+      description: fasta/fastq file
+      pattern: "*.{fasta,fastq}"
```

Comment:
mode & genomesize are missing a documentation?
especially those two input fields would benefit from some explanation I think

Reply:
Yes indeed, I added them later and forgot. Thanks!
```
  - mode:
      type: value
      description: Canu mode depending on the input data (source and error rate)
      pattern: "-pacbio|-nanopore|-pacbio-hifi"
  - genomesize:
      type: value
      description: An estimate of the size of the genome. Common suffices are allowed, for example, 3.7m or 2.8g
      pattern: "<number>[g|m|k]"
```